pkg/views/workspace/create/configuration.go

```diff
@@ -179,7 +189,7 @@ func GetProjectConfigurationForm(projectConfiguration *ProjectConfigurationData)
 				if s == "" {
 					return errors.New("devcontainer file path is required")
 				}
-				return nil
+				return validateDevcontainerFilename(s)
```

Comment:
This should just be `.Validate(validateDevcontainerFilename)` as the empty check is redundant

Reply:
Makes sense. Could you check now?